tests/Integration/Behaviour/Features/Context/ShopFeatureContext.php

```diff
@@ -65,16 +73,169 @@ public function shopWithNameExists(string $reference, string $shopName)
     /**
      * @Given single shop context is loaded
      */
-    public function singleShopContextIsLoaded()
+    public function singleShopContextIsLoaded(): void
     {
         Shop::setContext(Shop::CONTEXT_SHOP, Configuration::get('PS_SHOP_DEFAULT'));
     }
 
     /**
      * @Given multiple shop context is loaded
      */
-    public function multipleShopContextIsLoaded()
+    public function multipleShopContextIsLoaded(): void
     {
         Shop::setContext(Shop::CONTEXT_ALL);
     }
+
+    /**
+     * @Given I add a shop group :reference with name :groupName and color :color
+     *
+     * @param string $reference
+     * @param string $groupName
+     * @param string $color
+     */
+    public function addShopGroup(string $reference, string $groupName, string $color): void
```

Comment:
You will probably have conflicts with this pending PR from 177 https://github.com/PrestaShop/PrestaShop/pull/22228

Either you or when we merge it to develop, maybe we should wait for it to be merged in 177 and then back to develop, it will be easier to solve conflicts on this PR than during the merge which will lack context

Reply:
Nevermind it's the same code copied anyway ^^